src/status_im/chat/models/message_list.cljs

```diff
@@ -116,11 +118,11 @@
     :as   previous-message}]
   (let [last-in-group? (not (same-group? current-message previous-message))]
     (assoc previous-message
-           :display-username? (and last-in-group?
-                                   (not system-message?)
-                                   (not outgoing)
-                                   (not one-to-one?))
-           :last-in-group?    last-in-group?)))
+      :display-username? (and last-in-group?
+                              (not system-message?)
+                              (not outgoing)
+                              (not one-to-one?))
+      :last-in-group? last-in-group?)))
```

Comment:
Seems vertical aligning got reverted here

Reply:
I am facing a problem using Zprint with IntelliJ, also in this PR: https://github.com/status-im/status-mobile/pull/14631

Reply:
Could you give us more context on the problem @OmarBasem? `zprint` is an external tool, independent of IDE, so I'm genuinely curious about the problem. I know we have other IntelliJ users in the team, so maybe it's worth asking.

Reply:
@ilmotta when I run `make lint-fix` or even this (https://github.com/status-im/status-mobile/pull/14589#issuecomment-1359515798) it keeps messing up the formatting of other files/code which is not part of the PR. I am not sure if this has to do with the IDE. Also there is a discussion here https://github.com/status-im/status-mobile/pull/14631/#pullrequestreview-1229075222

Reply:
That's currently what `make lint-fix` is supposed to do, it fixes everything :shrug: We could have a separate make target that only formats what's changed in the branch under development or something like that, maybe that's going to be better for you.

The current `lint-fix` target collects files using:

```shell
git ls-files | grep -e '\.clj$' -e '\.cljs$' -e '\.cljc$' -e '\.edn$'
```

This returns all files. Check in your machine to be sure.

I'm not sure about the best approach for IntelliJ users, but relying on `make lint-fix` is supoptimal IMO. Perhaps you and @qfrank would have a better experience using a git hook that only formats the files you care about.